### PR TITLE
web/admin: fix footer links not being parsed on settings page

### PR DESCRIPTION
**Pull request by rissson**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: web/src/admin/admin-settings/AdminSettingsForm.ts [hunk 1]
```diff
@@ -156,8 +157,7 @@ export class AdminSettingsForm extends Form<SettingsRequest> {
             <ak-form-element-horizontal label=${msg("Footer links")} name="footerLinks">
                 <ak-codemirror
                     mode=${CodeMirrorMode.YAML}
-                    .parseValue=${false}
-                    value="${first(this._settings?.footerLinks, [])}"
+                    value="${YAML.stringify(first(this._settings?.footerLinks, []))}"
```
Comment (BeryJu) [inline web/src/admin/admin-settings/AdminSettingsForm.ts:160]:
```suggestion
                    .value="${first(this._settings?.footerLinks, [])}"
```

with that the element will do the parsing itself

Comment (BeryJu) [reply web/src/admin/admin-settings/AdminSettingsForm.ts:160]: also on L164 could you change it to say JSON or YAML

Comment (rissson) [reply web/src/admin/admin-settings/AdminSettingsForm.ts:160]: With JSON and no parsing, it shows up as 
image

Comment (rissson) [reply web/src/admin/admin-settings/AdminSettingsForm.ts:160]: Same with YAML.

Comment (rissson) [reply web/src/admin/admin-settings/AdminSettingsForm.ts:160]: I didn't, that might be it

Comment (rissson) [reply web/src/admin/admin-settings/AdminSettingsForm.ts]: Yep, it indeed does the trick. I left it as YAML though, as with JSON it still shows up as `[object Object]`

**Pull request merged**